onset_maproom/calc.py

```diff
@@ -1,16 +1,295 @@
 import numpy as np
 import pandas as pd
 import xarray as xr
+#from pint import application_registry as ureg
+#import cf_xarray.units
 
-# Date Reading functions
+# Data Reading functions
 
 
 def read_zarr_data(zarr_path):
     zarr_data = xr.open_zarr(zarr_path)
     return zarr_data
 
 
-# Growing season functions
+# Water Balance functions
```

Comment:
I split it off into an agronomy.py and its test file. I didn't do it for test data because some are used by both. Seemed more arbitrary to pick which test data goes where (as surely we don't want to duplicate, do we?). Let me know

Reply:
And actually I wanted to rebase and test that the older water_balance function is used by the app from agronomy rather than calc... but forgot to do that...

Reply:
This conversation is resolved as far as I'm concerned, but I'll leave it open because your last comment is a reminder to yourself. Feel free to resolve when it's no longer needed.